From e3f3e266249f77ff655299daeab3128347d6cb17 Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Wed, 30 Jan 2013 15:59:40 +0100
Subject: [PATCH] patch: desktop-file

Squashed commit of the following:

commit e472b870c4490f41b9257c835d4c8c72a575e9e9
Author: Nils Philippsen <nils@redhat.com>
Date:   Wed Jan 30 15:57:57 2013 +0100

    desktop file: use Name, GenericName, X-GNOME-FullName

commit 9f7f6a039193f91473ded79780bd72e29d7b94fb
Author: Nils Philippsen <nils@redhat.com>
Date:   Wed Jan 30 15:57:14 2013 +0100

    desktop file: remove obsolete encoding key

commit 79a444793a60bd729c72283ad1920f0ce9c65dc2
Author: Nils Philippsen <nils@redhat.com>
Date:   Fri Nov 19 12:41:23 2010 +0100

    customize desktop file
---
 src/xsane.desktop | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/src/xsane.desktop b/src/xsane.desktop
index d5161e5..a2a4a61 100644
--- a/src/xsane.desktop
+++ b/src/xsane.desktop
@@ -1,9 +1,14 @@
 [Desktop Entry]
-Encoding=UTF-8
-Name=XSane - Scanning
+Version=1.0
+#Name=XSane - Scanning
+Name=XSane
+GenericName=Scanner Tool
+X-GNOME-FullName=XSane (Scanner Tool)
 Comment=Acquire images from a scanner
 Exec=xsane
+TryExec=xsane
 Icon=xsane
 Terminal=false
 Type=Application
-Categories=Application;Graphics
+Categories=Graphics;2DGraphics;RasterGraphics;Scanning;GTK;
+StartupNotify=true
-- 
1.8.1

